sources/packages/web/src/components/institutions/modals/AddInstitutionUserModal.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <!-- Add user -->
+  <Dialog
+    v-if="showAddUser"
+    header="Add User to Account"
+    v-model:visible="display"
```

Comment:
Please consider removing the v-model to be in alignment with the other bindings.